src/controllers/controller.line.js

```diff
@@ -315,4 +315,19 @@ module.exports = DatasetController.extend({
 		model.borderWidth = valueOrDefault(options.hoverBorderWidth, options.borderWidth);
 		model.radius = valueOrDefault(options.hoverRadius, options.radius);
 	},
+
+	setDatasetHoverStyle: function() {
+		var line = this.getMeta().dataset;
+		var model = line._model;
+		var config = this._config;
+		var getHoverColor = helpers.getHoverColor;
+
+		line.$previousStyle = {
+			backgroundColor: model.backgroundColor,
```

Comment:
This seems like a small set of the available properties. Should we support more of the properties defined here: https://www.chartjs.org/docs/latest/charts/line.html#dataset-properties ? Or at least `borderWidth` since it's supported in `setHoverStyle`?

Reply:
Added `hoverBorderWidth` and documentation about available hover* options.